src/sentry/static/sentry/app/views/organizationEventsV2/data.jsx

```diff
@@ -73,6 +92,33 @@ export const ALL_VIEWS = deepFreeze([
  * displays with a custom render function.
  */
 export const SPECIAL_FIELDS = {
+  '% query': {
+    fields: [],
+    sortField: false,
+    renderFunc: () => {
+      return <Container>% query</Container>;
```

Comment:
Do we want a `t()` here?

Reply:
I'll remove this column formatter as it isn't being used right now.